pdc_client/tests/component/data/global_component/detail.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "1",
-    "name": "Test Global Component",
+    "name": "Test_Global_Component",
```

Comment:
Because when run the command :" self.runner.run(['global-component', 'update', 'Test_Global_Component', '--name', 'new_test_name']),  " ;  
which corresponding to the pdc comand: "pdc  global-component  update  --name new_test_name  Test_Global_Component ". If global-component's name is "Test Global Component",  which exist blank space, the pdc command will split it to 3 parts,  and fail to filter the name.